config/config.inc

```diff
@@ -49,6 +49,15 @@ if [ -n "$CUDNN_DIR" ]; then
   SET_CUDNN="-DCUDNN_PATH=${CUDNN_DIR}"
 fi
 
+# build legion only 
+if [ "$BUILD_LEGION_ONLY" = "ON" ]; then
+  SET_BUILD_LEGION_ONLY="-DBUILD_LEGION_ONLY=ON"
+elif [ "$FF_USE_PYTHON" = "OFF" ]; then
+  SET_BUILD_LEGION_ONLY="-DBUILD_LEGION_ONLY=OFF"
+else
+  SET_BUILD_LEGION_ONLY="-DBUILD_LEGION_ONLY=ON"
```

Comment:
by default (if BUILD_LEGION_ONLY is not set in the config.linux), we should set this value to OFF (`-DBUILD_LEGION_ONLY=OFF`)